app/views/users/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Редактирование профиля</h1>
+
+<div class="row">
+  <div class="col-md-6 buffer-top">
+
+    <%= render 'form' %>
+
+    <p class="buffer-top">
+      <%= link_to 'Сменить пароль', edit_user_registration_path %>
+    </p>
+
+  </div>
+</div>
+
+<%= link_to 'Show', @user %> |
+<%= link_to 'Back', users_path %>
```

Comment:
It would be great if you install gem 'slim-rails' and re-create all .erb views using SLIM